docs/Previous_Changelogs.inc

```diff
@@ -0,0 +1,9 @@
+Previous Changelogs
+-------------------
+
+.. toctree::
+   :maxdepth: 1
+
+   previous_changelogs/6.1.0_Changelog
```

Comment:
This should be changed to a glob so that it just picks them all up as we add them to the directory.

Reply:
Glob sorts them in the wrong order (oldest to newest).  The 'reversed' parameter doesn't work with the version of sphinx we are using.